openc3-cosmos-init/plugins/packages/openc3-cosmos-tool-cmdtlmserver/src/tools/CmdTlmServer/RawDialog.vue

```diff
@@ -338,15 +337,17 @@ export default {
   border-color: white;
   resize: both;
   overflow: auto;
+  min-height: 28px;
   max-height: 85vh;
+  min-width: 800px;
```

Comment:
I think this needs to be 830 ... try opening the INST IMAGE packet with all the ascii